client/src/components/EventView/RSVPFloating/index.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
```

Comment:
You can delete this file. The component already exists and is called "ActionPrompt"